lib/lpc55-rot-startup/src/lib.rs

```diff
@@ -60,6 +63,22 @@ fn apply_memory_protection(mpu: &MPU) {
     }
 }
 
+// Execute this function before jumping to Hubris.
+// This function will panic if PUF is not in the desired state.
+#[cfg(any(feature = "dice-mfg", feature = "dice-self"))]
+fn puf_check(puf: &lpc55_pac::PUF) {
+    use crate::dice::KEY_INDEX;
+    let puf = Puf::new(puf);
+
+    if !puf.is_index_blocked(KEY_INDEX).unwrap_lite()
```

Comment:
I had to read this twice to realize blocked and locked were different.  Maybe `is_index_blocked` and `is_index_blocking_locked`?